src/nvim/testdir/test_menu.vim

```diff
@@ -1,9 +1,32 @@
 " Test that the system menu can be loaded.
 
+if !has('menu')
+  finish
+endif
+
 func Test_load_menu()
   try
     source $VIMRUNTIME/menu.vim
   catch
     call assert_report('error while loading menus: ' . v:exception)
   endtry
+  call assert_match('browse confirm w', execute(':menu File.Save'))
```

Comment:
This test is failing.

Nvim seems to have a bug where `:menu File.Save` does not print the menu definition. Only `:menu File` works. Need to fix this bug.